plugins/kubernetes/app/models/kubernetes/deploy_yaml.rb

```diff
@@ -47,6 +51,41 @@ def template
       end
     end
 
+    # process the annotations, and expand $ENV and $DEPLOY_GROUP for any annotation
```

Comment:
`process/handle/manage` all the same empty fill works :D 

this reads nicely :)
`# expand $ENV and $DEPLOY_GROUP in annotation that start with 'secret/'